Implement page-up and page-down commands

ktomsic opened the pull request:
These are just mapped to 'pu' / 'page-up' and 'pd' / 'page-down'.

ktomsic committed:
Implement page-up and page-down commands

These are just mapped to 'pu' / 'page-up' and 'pd' / 'page-down'.

ktomsic commented:
@bytebuddha: 
How many commands per PR seems reasonable to you? I figured I'd keep things short and simple to keep it easy to review, but I also don't want to flood you with small PRs.

For example, would you prefer the move-* (and possibly the select-*) commands added in here, or as a separate PR?

bytebuddha commented:
I guess i'd prefer smaller, say 2-6 commands at a time because there easier to review, but i appreciate the work your putting in on this and i'll be glad to review any PR your willing to make :smile:.

bytebuddha commented:
@ktomsic this looks really good i only have one nit

In View::handle_input(line: 144-145) we are calling client::page_up() & Client::page_down() directly, can you change those to call the new View::page_up & View::page_down functions?

ktomsic committed:
view: Switch handle_input() from client calls to View:: functions

The commandline handler uses View:: wrappers around client calls, while
View::handle_input() calls into the client directly. Since the keyboard
shortcuts should be identical to commandline commands, this changeset
tweaks handle_input() to call the wrappers. This way, we can update the
command and keyboard handlers but just tweaking the wrapper.
